packages/features/bookings/lib/getAllCredentialsForUsersOnEvent/getAllCredentials.test.ts

```diff
@@ -0,0 +1,699 @@
+import { describe, test, expect, vi } from "vitest";
+
+import { UserRepository } from "@calcom/lib/server/repository/user";
+import {
+  createCredentials,
+  addTeamsToDb,
+  addEventTypesToDb,
+  addUsersToDb,
+} from "@calcom/web/test/utils/bookingScenario/bookingScenario";
+
+// vi.mock("@calcom/lib/server/repository/user", () => {
+//   return {
+//     enrichUserWithItsProfile
+//   }
+// })
+
+describe("getAllCredentials", () => {
+  test("Get an individual's credentials", async () => {
+    vi.spyOn(UserRepository, "enrichUserWithItsProfile").mockReturnValue({
+      profile: null,
+    });
+
+    const getAllCredentials = (await import("./getAllCredentials")).getAllCredentials;
+
+    const userCredential = {
+      id: 1,
+      type: "user-credential",
+      userId: 1,
+      teamId: null,
+      key: {},
+      appId: "user-credential",
+      invalid: false,
+    };
+    await createCredentials([
+      userCredential,
+      { type: "other-user-credential", userId: 2, key: {} },
+      { type: "team-credential", teamId: 1, key: {} },
+    ]);
+
+    const credentials = await getAllCredentials(
+      {
+        id: 1,
+        username: "test",
+        credentials: [{ ...userCredential, user: { email: "test@test.com" } }],
+      },
+      {
+        userId: 1,
+        team: null,
+        parentId: null,
+        metadata: {},
+      }
+    );
+
+    expect(credentials).toHaveLength(1);
+
+    expect(credentials).toContainEqual(expect.objectContaining({ type: "user-credential" }));
+  });
+
+  describe("Handle CRM credentials", () => {
+    describe("If CRM is enabled on the event type", () => {
+      describe("With _crm credentials", () => {
+        test("For users", async () => {
+          vi.spyOn(UserRepository, "enrichUserWithItsProfile").mockReturnValue({
+            profile: null,
+          });
+
+          const getAllCredentials = (await import("./getAllCredentials")).getAllCredentials;
```

Comment:
This should be mocked i think?

Reply:
Or maybe not we just have instances of prisma being called in here when teamId is present